packages/pwa/app/pages/account/wishlist/partials/wishlist-secondary-button-group.jsx

```diff
@@ -68,7 +81,10 @@ const WishlistSecondaryButtonGroup = ({productListItemId, onClick = noop}) => {
                     onClick={showRemoveItemConfirmation}
                     data-testid={`sf-wishlist-remove-${productListItemId}`}
                 >
-                    <FormattedMessage defaultMessage="Remove" />
+                    <FormattedMessage
+                        defaultMessage="Remove"
+                        id="wishlist_secondary_button_group.action.remove"
```

Comment:
Should we consider this as a button? 🤔  
```suggestion
                        id="wishlist_secondary_button_group.button.remove"
```

Reply:
Since this is inside `<Button variant="link">` that has an `onClick` prop, I'd consider this as an 'action'.